src/modules/tasks/tasks.service.ts

```diff
@@ -1,16 +1,45 @@
 import { Injectable, Logger } from '@nestjs/common'
 import { CommandBus } from '@nestjs/cqrs'
 import { Cron } from '@nestjs/schedule'
-import { PostFactorioCommand } from './commands/post-factorio.command'
+import { InjectRepository } from '@nestjs/typeorm'
+import { DeleteResult, Repository } from 'typeorm'
+import { RssFeedCommand } from './commands/rss-feed.command'
+import { Feed } from './entities/feed.entity'
 
 @Injectable()
 export class TasksService {
     private readonly logger = new Logger(TasksService.name)
 
-    constructor(private readonly commandBus: CommandBus) {}
+    constructor(@InjectRepository(Feed) private feedRepository: Repository<Feed>, private readonly commandBus: CommandBus) {}
 
     @Cron('0 * * * *')
     async handleCron() {
-        await this.commandBus.execute(new PostFactorioCommand())
+        await this.commandBus.execute(new RssFeedCommand())
```

Comment:
We should document what this does. Something like 'Every hour check RSS feeds."

Reply:
i thought everyone can read crontab like an analog clock, lol next pr i will